db/migrate/20150222172825_create_accounts.rb

```diff
@@ -7,7 +7,11 @@ def change
       t.string :oauth_access_token
       t.string :oauth_refresh_token
       t.datetime :oauth_expires_at
-      t.timestamps
+      t.text :synced_data
+      t.datetime :synced_all_at
+      t.string :email
```

Comment:
We dont need both `synced_data` and `email` columns.
Just using `synced_data` covers everything and the `synced` statement becomes `synced id_key: :uid, delegate_attributes: [:email]`